tests/driver_at86rf2xx_aes/Makefile

```diff
@@ -0,0 +1,23 @@
+include ../Makefile.tests_common
+
+DISABLE_MODULE += auto_init_at86rf2xx
+
+# define the driver to be used for selected boards
+ifneq (,$(filter samr21-xpro,$(BOARD)))
+  DRIVER := at86rf233
+endif
+ifneq (,$(filter iotlab-m3 fox,$(BOARD)))
+  DRIVER := at86rf231
+endif
+ifneq (,$(filter mulle,$(BOARD)))
+  DRIVER := at86rf212b
+endif
+
+# use the at86rf231 as fallback device
+DRIVER ?= at86rf231
+
+# include the selected driver
+USEMODULE += $(DRIVER)
```

Comment:
Do we actually need the base driver here?

Reply:
for this in main.c:
```C
    at86rf2xx_setup(&dev, &at86rf2xx_params[0]);
    dev.netdev.netdev.event_callback = _event_cb;
    if (dev.netdev.netdev.driver->init(&dev.netdev.netdev) != 0) {
        return EXIT_FAILURE;
    }
```

Reply:
Yea but do we need it?
I mean if we have to init the radio to use crypto that's fine.
But if we just init it and then never use it, we might as well drop it.

Reply:
I don´t think we need the driver capability but it initializes the `params`,  `SPI` and checks if SPI is working.
You mean I can copy setup the params and init SPI manually? Then the driver is not needed. Is this better?

Reply:
Then I would have to make an own driver rather then a pseudomodule, right?

Reply:
A never mind then.
A separate init function would be possible, but that's not worth the hassle now.

If someone really wants to use the crypto without the radio, that could be added later.